pkg/vm/builtins/range_check.go

```diff
@@ -0,0 +1,31 @@
+package builtins
+
+import (
+	"fmt"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+type RangeCheck struct{}
+
+// 1 << 128
+var max, _ = new(fp.Element).SetString("0x100000000000000000000000000000000")
```

Comment:
I would directly initialize max in montgomery form. I would also call it `max128` or another name that you could infer it is a 2**128.

```suggestion
var max = fp.Element{
            8446744073700081665,
            17407,
            18446744073709551584,
            576460752142434320,
      }
```